src/control/AxesController.ts

```diff
@@ -214,6 +214,39 @@ class AxesController {
     return this;
   }
 
+  /**
+   * Change the destination and duration of the animation currently playing
+   * @ko 재생 중인 애니메이션의 목적지와 재생 시간을 변경합니다
+   * @param {number} position A position to move<ko>이동할 좌표</ko>
+   * @param {number} duration Duration of the animation (unit: ms)<ko>애니메이션 진행 시간 (단위: ms)</ko>
+   * @chainable
+   * @return {this}
+   */
+  public updateAnimation(position: number, duration?: number): this {
+    this._animatingContext = {
+      ...this._animatingContext,
+      end: position
+    };
+    this._axes?.updateAnimation({
+      destPos: { [AXES.POSITION_KEY]: position },
+      duration
```

Comment:
As changing `duration` in Flicking does not directly change Axes's option `duration`, you should use Flicking's default duration as a fallback.

```js
duration: duration ?? this._flicking.duration
```

Reply:
`updateAnimation` of the Axes keeps the duration of the current animation when the [duration does not exist.](https://github.com/naver/egjs-axes/blob/master/packages/axes/src/animation/EasingManager.ts#L29)
However, I found that replacing the || with ?? in the code linked above works correctly when the new duration is 0.
This will be fixed in Axes' recent PR.